server/settings/environments/development.py

```diff
@@ -15,6 +15,9 @@
 
 DEBUG = True
 
+# Disable axis in development (which is also used for testing!)
+AXES_ENABLED = False
```

Comment:
`AXES` is for checking suspicious logins, which only applies to production. In development/tests needs to be disabled in order to allow test runners to work.